corehq/apps/es/client.py

```diff
@@ -1318,23 +1318,50 @@ def create_document_adapter(cls, index_name, type_, *, secondary=None):
     """Creates and returns a document adapter instance for the parameters
     provided.
 
+    One thing to note here is that the behaviour of the function can be altered with django settings.
+    If ES_<app_name>_INDEX_SWAPPED is set to True, then primary index will act as secondary index and vice versa.
+    This will be ignored if ES_<app_name>_INDEX_MULTIPLEXED is set to True
```

Comment:
What does "This" refer to? It seems like it might be referring to `ES_<app_name>_INDEX_SWAPPED`, but then I would expect a different value:
```suggestion
    ES_<app_name>_INDEX_SWAPPED will be ignored if ES_<app_name>_INDEX_MULTIPLEXED is set to False
```

Reply:
Yeah. Your suggestion makes perfect sense. Thanks for catching that ⭐